flexdata/data.py

```diff
@@ -435,14 +457,38 @@ def read_flexraylog(path, sample = 1):
     geom.parameters['det_pixel'] *= sample
     geom.parameters['img_pixel'] *= sample
 
-    # Some Flexray scanner-specific motor offset corrections:
-    _flex_motor_correct_(geom)
+    geom = correct_roi(geom)
+
+    if sample != 1:
+        msg = f"Adjusted geometry by binning by {sample}"
+        logging.info(msg)
+        geom.log(msg)
 
     return geom
 
-def read_flexraymeta(path, sample = 1):
+
+def read_flexraymeta(*args):
```

Comment:
This function is still called by `read_flexray`.

Reply:
Yes, but it only displays a deprecation warning. A function parse_flexraymeta that parses a file has been added.